packages/xo-server/src/xapi/index.js

```diff
@@ -2348,4 +2348,31 @@ export default class Xapi extends XapiBase {
       )
     }
   }
+
+  async isHyperthreading(hostId) {
+    try {
+      const hyperthreading = await this.call(
+        'host.call_plugin',
+        this.getObject(hostId).$ref,
+        'hyperthreading.py',
+        'get_hyperthreading',
+        {}
+      )
+      if (hyperthreading === 'false') {
```

Comment:
```js
return  hyperThreading === 'true'
```

Reply:
I think that we need to very precise here, because if hyperthreading is different than `true` and `false` we should throw.
@julien-f what do you think about this ?

Reply:
We know how the plugin is implemented therefore I think we can keep things simple.

And, to be on the safe side, testing `'false'` is preferable because we prefer false positive to false negative: `true` is the problematic value and we don't want to miss it.